internal/mode/static/nginx/config/servers.go

```diff
@@ -543,9 +546,13 @@ func createProxySSLVerify(v *dataplane.VerifyTLS) *http.ProxySSLVerify {
 	}
 }
 
-func createReturnValForRedirectFilter(filter *dataplane.HTTPRequestRedirectFilter, listenerPort int32) *http.Return {
+func createReturnValForRedirectFilter(
```

Comment:
Can you update the function name to reflect its new purpose? It creates more than a return val.